src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int INCREASE_FACTOR = 2;
+    private static final double LOAD_MAX_PERCENT = 0.75;
+    private static final int ZERO = 0;
+
+    private Node<K,V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[INITIAL_CAPACITY];
+        threshold = (int)(INITIAL_CAPACITY * LOAD_MAX_PERCENT);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > threshold) {
+            resize();
+        }
+        if (putToArray(key, value, table)) {
+            size++;
+        }
     }
 
     @Override
     public V getValue(K key) {
+        int hash;
+        if (key == null) {
+            hash = ZERO;
+        } else {
+            hash = getHashCode(key);
+        }
+        int index = hash % table.length;
```

Comment:
You can simplify this by creating a simple method with ternary operator to avoid repetition in code